contest/contest_info.py

```diff
@@ -108,17 +110,59 @@ def get_clarifications(user,contest):
         return reply_all | user_ask
     return reply_all
 
-def is_contestant(user,contest):
+def is_contestant(user, contest):
     if user.is_authenticated():
         contestant = Contestant.objects.filter(contest = contest, user = user)
         if len(contestant) >= 1:
```

Comment:
Use ``if contestant:`` as it is suggested by style guide.

https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=True/False_evaluations#True/False_evaluations